src/Tools/GenerateProps/targetgroups.props

```diff
@@ -4,181 +4,212 @@
     <TargetGroups Include="netcore50">
       <TargetingPackNugetPackageId>Microsoft.TargetingPack.Private.NetNative</TargetingPackNugetPackageId>
       <NuGetTargetMoniker>.NETCore,Version=v5.0</NuGetTargetMoniker>
+      <NuGetTargetMonikerShort>netcore50</NuGetTargetMonikerShort>
```

Comment:
Given you have these here why do you have them in all the project files as well?